test/fakeintake/aggregator/traceAggregator.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package aggregator
+
+import (
+	"time"
+
+	pb "github.com/DataDog/datadog-agent/pkg/proto/pbgo/trace"
+	"github.com/DataDog/datadog-agent/test/fakeintake/api"
+
+	"google.golang.org/protobuf/proto"
+)
+
+// TracePayload is a payload type for traces
+type TracePayload struct {
+	*pb.AgentPayload
+	collectedTime time.Time
+}
+
+// name returns the hostname of the agent that produced the payload
+func (tp *TracePayload) name() string {
+	return tp.HostName
+}
+
+// GetTags is not implemented
```

Comment:
Same here, we should mention `PayloadItem`.

Reply:
Done in https://github.com/DataDog/datadog-agent/pull/22003/commits/30953ae77d821cd8a1fc5d742c70854d478ec5da